app/import_functions.go

```diff
@@ -1580,9 +1581,18 @@ func (a *App) importEmoji(data *EmojiImportData, dryRun bool) *model.AppError {
 
 	var emoji *model.Emoji
 
-	emoji, appError := a.Srv().Store.Emoji().GetByName(*data.Name, true)
-	if appError != nil && appError.StatusCode != http.StatusNotFound {
-		return appError
+	emoji, err := a.Srv().Store.Emoji().GetByName(*data.Name, true)
+	if err != nil {
+		var nfErr *store.ErrNotFound
+		var appErr *model.AppError
+		switch {
+		case errors.As(err, &nfErr):
```

Comment:
We can make the check in the error check itself by checking for `!errors.As(err, &nfErr)` and then moving to this switch-case.